tests/assets/config.yaml

```diff
@@ -1,3 +1,12 @@
-stages:
-  network:
-    - hostname: "testhostname"
\ No newline at end of file
+install:
```

Comment:
ditto, the new layout should be documented

Reply:
(this is particularly relevant for the harvester team, as they use this feature) @bk201